packages/shared/styles/components/SfScrollable.scss

```
@@ -0,0 +1,25 @@
+@import "~simplebar/dist/simplebar.min.css";
+@import "../variables";
+
+$scrollable__max-width: 13.5rem !default;
+
+.sf-scrollable {
+    --content-height: 2rem;
```

```suggestion
    --content-height: 10rem;
```
we can set the highest value